src/block.js

```diff
@@ -0,0 +1,9 @@
+import { defaultConfig } from './blockConfig';
+
+export const factory = React => {
```

Comment:
I really hate having the factory function being called just "factory". I'd rather it be named after the block/component, even if you do need to update it in `src/index.js` when you change it. There's too many factory functions. It gets confusing. I would suggest `StarterBlockFactory` instead.